st2tests/st2tests/fixtures/packs/BUILD

```diff
@@ -22,6 +23,7 @@ shell_sources(
 
 python_sources(
     name="test_content_version",
+    # do not fmt across git submodule boundary
     skip_black=True,
```

Comment:
This was requested in https://github.com/StackStorm/st2/pull/5774#discussion_r997170988
I missed adding it before merging, so I'm adding it here.